packages/core/src/log.ts

```diff
@@ -0,0 +1,38 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the MIT License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019-Present Datadog, Inc.
+
+import c from 'chalk';
+
+import type { LogLevel } from './types';
+
+const log = (text: string, level: LogLevel, type: LogLevel = 'debug', name?: string) => {
+    let color = c;
+    // eslint-disable-next-line no-console
```

Comment:
You should disable this rule fully in this file with `// eslint-disable no-console` at the top and remove all the inline ones